README.md

```diff
@@ -72,3 +72,9 @@ This will build the app in production mode and save it into the `.next` folder.
 best performance, minimized and the filenames include a hash value of the content. Don't edit them by hand!
 
 You can run the production build using the built-in server with `yarn start`.
+
+## UI Test
+
+Curious about the new look and feel? We provide a demo of the new UI on [hedgedoc.dev](https://hedgedoc.dev). This version uses mocked data and has no data persistence.
```

Comment:
```suggestion
Curious about the new look and feel? We provide a demo of the new UI on [hedgedoc.dev](https://hedgedoc.dev). This version uses mocked data and has no data persistence.
The UI demo is hosted on Netlify, their [privacy policy](https://www.netlify.com/privacy) applies.
```
Maybe also add a note to https://hedgedoc.org/privacy-policy/?

Reply:
I like the idea of adding the netlify privacy link. However, I don't see how this is related to hedgedoc.org/privacy-policy.

Reply:
tbh I would just link our privacy policy and add netlify to it. I mean. We list every service we use there.

Reply:
See https://github.com/hedgedoc/hedgedoc.github.io/pull/202